features/step_definitions/modal_steps.rb

```diff
@@ -1,3 +1,12 @@
-Then /I should see the mention modal/ do
+Then /^I should see a modal$/ do
+  step %{I should see a ".modal.in"}
+end
+
+Then /^I should see the mention modal$/ do
   step %{I should see a "#mentionModal.in"}
 end
+
+When /^I put in my password in the close account modal$/ do
+  execute_script("$(\"#closeAccountModal input#close_account_password\").val(\"#{@me.password}\")")
```

Comment:
Please add a comment explaining why the `execute_script` solution is used instead of the Capybara helper.